src/Storages/ExternalStream/Kafka/Kafka.h

```diff
@@ -52,6 +57,7 @@ class Kafka final : public StorageExternalStreamImpl
     Poco::Logger * log;
 
     NamesAndTypesList virtual_column_names_and_types;
+    klog::KConfParams parsed_properties;
```

Comment:
nit : Rename as `properties`

Reply:
There is a public method call `properties`. I can name this field `properties_` instead. Does it sound right?

Reply:
Then we are good to go without changes, or rename it to kafka_properties